public/css/default.css

```diff
@@ -0,0 +1,11 @@
+footer a {
+    color: #92bfec;
+}
+a .badge-dark {
+    text-decoration: none;
+    color: #fff;
+}
+
+.mw-40ch {
```

Comment:
Completely presentational class name that could probably just be replaced with an inline `style=` attribute.  Not super worried about it, though.

Also TIL about `ch`.  Why back when I was a frontend dev, we had `em`, and we liked it.